hijack/helpers.py

```diff
@@ -128,15 +139,24 @@ def login_user(request, hijacked):
     backend = get_used_backend(request)
     hijacked.backend = "%s.%s" % (backend.__module__, backend.__class__.__name__)
 
+    pre_hijack_started_results = pre_hijack_started.send(
```

Comment:
Though not forbidden, it is discouraged to use signal results in Django. Might I ask why need the result and also want to pass it to the next signal?
```suggestion
    pre_hijack_started.send(
```